src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/DSA.Xml.cs

```diff
@@ -127,10 +127,12 @@ public override string ToXmlString(bool includePrivateParameters)
             {
                 if (keyParameters.X == null)
                 {
+#pragma warning disable CA2208 // Instantiate argument exceptions correctly
                     // .NET Framework compat when a 3rd party type lets X be null when
                     // includePrivateParameters is true
                     // (the exception would have been from Convert.ToBase64String)
                     throw new ArgumentNullException("inArray");
+#pragma warning restore CA2208
```

Comment:
> .NET Framework compat

@bartonjs, any reason we can't fix the argument name here?

Reply:
As per the previous comments, i have updated argument names even if there platform compatibility warning comments.  but i didn't changed this one because the condition/situation is not related to the arguments, it might better to use other exceptions though, so such situations (there is few more)

Reply:
Yeah, this should be an InvalidOperationException (had ExportParameters used the template method pattern, the exception ideally would be in the non-virtual call as a post-virtual verification).  Just suppressing it seems OK, it's purely legacy code.